components/alert.scss

```diff
@@ -0,0 +1,9 @@
+$fd-icons-path: "../scss/icons/";
+$fd-fonts-path: "../scss/fonts/";
+
+@import "../scss/components/alert";
+@import "../scss/components/link";
+@import "../scss/core/elements";
+@import "../scss/core/root";
```

Comment:
Importing the entire core elements and root files is not what we want to do here. We need to find out what from these files is necessary to render alerts properly and bring those into the link class definitions -- either via common mixins or directly. The whole point of self-contained styles is not only bring along what is needed and not do the changes using elements.